src/main/java/mate.academy/Main.java

```diff
@@ -4,13 +4,13 @@
 import mate.academy.lib.Injector;
 import mate.academy.model.Product;
 import mate.academy.service.ProductService;
+import mate.academy.service.impl.ProductServiceImpl;
 
 public class Main {
-
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductServiceImpl)
+                injector.getInstance(ProductService.class);
```

Comment:
It's not necessary to cast the result of `injector.getInstance(ProductService.class)` to `ProductServiceImpl`. Since `getInstance` will return an instance of `ProductService`, you can directly assign it without casting. Casting here is redundant and can lead to a `ClassCastException` if the implementation of `ProductService` changes in the future.